sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample1_DetectEntireSeriesAnomaly.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.AI.AnomalyDetector.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.AnomalyDetector.Tests.Samples
+{
+    public partial class AnomalyDetectorSamples : RecordedTestBase<AnomalyDetectorTestEnvironment>
+    {
+        public AnomalyDetectorSamples(bool isAsync) : base(isAsync)
+        {
+            Sanitizer = new AnomalyDetectorRecordedTestSanitizer();
+        }
```

Comment:
You wouldn't need this ctor if you inherit from SamplesBase.

Reply:
Removed.

---
In reply to: [483317221](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r483317221) [](ancestors = 483317221)